src/app/routes/index.js

```diff
@@ -13,10 +19,23 @@ import {
   radioAndTvPath,
 } from './regex';
 
+// CPS Asset Mapping to PageType
 const CpsAsset = props => {
-  const type = pathOr('STY', ['pageData', 'metadata', 'type'], props);
-  const Page = type === 'FIX' ? FrontPage : CpsAssetPage;
-  return Page({ ...props, pageType: type });
+  const type = path(['pageData', 'metadata', 'type'], props);
+
+  switch (type) {
+    case 'STY':
+      return CpsSty({ ...props, pageType: type });
+    case 'PGL':
+      return CpsPgl({ ...props, pageType: type });
+    case 'MAP':
+      return CpsMap({ ...props, pageType: type });
+    case 'FIX': // TODO: Create FIX Page if required
+      return FrontPage({ ...props, pageType: type });
```

Comment:
As an aside, do you think it'd make sense to maintain a common module in which we export a list of page type constants to aid readability and reduce the chance of fragmentation? It could also be used here: https://github.com/bbc/simorgh/blob/latest/src/app/routes/getInitialData/utils/getPreprocessorRules/index.js#L8-L14.

Reply:
ohh nice, I did not know we had those, will update this PR. Thanks @simonsinclair

Reply:
I think a new PR to move pageHandlers out of containers and put them at the same dir level is good idea. Similar to what Andrew mentioned earlier.

Reply:
I was making a case _against_ moving them, but can see both sides of the argument. 😄

Reply:
made that change @simonsinclair